.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: ruby
+os: osx
+env: OSX=10.12
```

Comment:
@mistydemeo Travis CI has limited numbers of workers running at once per org and a generally limited number of macOS workers. As a result I think it's better sticking with a single version but I'm 👍 to this being an older one. Just wanted to 🚢 this to test the other PRs in progress. Feel free to change in a new PR 👍